src/Riok.Mapperly.Abstractions/MapperNestedPropertiesAttribute.cs

```diff
@@ -0,0 +1,37 @@
+namespace Riok.Mapperly.Abstractions;
+
+/// <summary>
+/// Maps all properties from a nested path to the root destination.
+/// </summary>
+[AttributeUsage(AttributeTargets.Method, AllowMultiple = true)]
+public class MapperNestedPropertiesAttribute : Attribute
```

Comment:
Could you rename according to the issue and other mapping related attributes to `MapNestedPropertiesAttribute`.